test/commands_on_geo_test.rb

```diff
@@ -12,6 +12,21 @@ def setup
     end
   end
 
+  def test_geoadd_with_array_params
+    target_version "4.1.1" do
```

Comment:
This is not what `target_version` is for.

It's there to specify the minimal redis server version, not the client one.